src/lib/jetlore.js

```diff
@@ -311,12 +311,11 @@ const getJetlore = () => {
       },
     
       _jsonp: function(self, url) {
-          var scriptTag = document.createElement("script");
-          scriptTag.type = "text/javascript";
-          scriptTag.src=url;
-          scriptTag.async = true;
-          document.getElementsByTagName("head")[0].appendChild(scriptTag);
-          self.log("SENT JSONP request: " + url)
+        var imgTag = document.createElement("img");
+        imgTag.src=url;
+        imgTag.style.display = 'none'
```

Comment:
in some browsers, display none might cause browser not to load it. Could you instead, style position fixed top left, with width and height of 1
https://stackoverflow.com/questions/12158540/does-displaynone-prevent-an-image-from-loading